optimum/exporters/openvino/model_configs.py

```diff
@@ -2894,6 +2894,12 @@ def patch_model_for_export(
         return super().patch_model_for_export(model, model_kwargs)
 
 
+@register_in_tasks_manager("minicpmo", *["image-text-to-text"], library_name="transformers")
+class MiniCPMOOpenVINOConfig(MiniCPMVOpenVINOConfig):
+    MIN_TRANSFORMERS_VERSION = "4.43.0"
+    MAX_TRANSFORMERS_VERSION = "4.51.0"
```

Comment:
```suggestion
    MAX_TRANSFORMERS_VERSION = "4.52.0"
```

I was able to export model having `transformers==4.51.3`. But in order for it to work `MAX_TRANSFORMERS_VERSION = "4.52.0"` needs to be set because there is a `>=` condition for raising and exception at https://github.com/huggingface/optimum-intel/blob/main/optimum/exporters/openvino/convert.py#L206 .

@echarlaix was this logic made this way on purpose, meaning that `MAX_TRANSFORMERS_VERSION` is an exclusive upper bound? I've just noticed it and in my opinion it is a bit confusing.

Is it common for transformers to do minor releases for older versions? If not, making the aforementioned condition a strict `>` would be more intuitive in my opinion. For this particular case we then could set `MAX_TRANSFORMERS_VERSION = "4.51.3"`.

Reply:
Hi @echarlaix! This is a gentle reminder. Could you please take a look at the comment above?

Reply:
@nikita-savelyevv, @echarlaix, just clarify - this is a separate unrelated story, right?:) We can address this issue in separate PR and discuss offline.

Reply:
we can use the `MAX_TRANSFORMERS_VERSION = "4.51.99"` notation

Reply:
@nikita-savelyevv yes this was done to enable all patch releases by default when setting the `MAX_TRANSFORMERS_VERSION` to 4.x.0 (as we don't know the number of patch release that will happen until the next minor release), but I understand that it might be a bit confusing so no problem to change it in a following PR if that makes things clearer

Reply:
I see, thanks for the explanation. Setting `MAX_TRANSFORMERS_VERSION = "4.51.99"` is ok with me. 

@rkazants Could you please update to 4.51.99?